drivers/lsm6dsxx/lsm6dsxx.c

```diff
@@ -71,7 +71,11 @@ int lsm6dsxx_init(lsm6dsxx_t *dev, const lsm6dsxx_params_t *params)
         return -LSM6DSXX_ERROR_BUS;
     }
 
+#ifdef LSM6DSXX_WHO_AM_I
```

Comment:
heh, the generic case

Reply:
Ah sorry, didn't see the ifdefs around.

Reply:
Why keep the non-generic case around at all?

Reply:
As mentioned above, it's kind of a hack to avoid having two different boards. Or is there any notion of board revision with different hardware in RIOT?